examples/ct/handlers.go

```diff
@@ -81,8 +82,11 @@ const (
 	GetEntryAndProofName  = EntrypointName("GetEntryAndProof")
 )
 
-// Entrypoints is a list of entrypoint names as exposed in statistics/logging.
-var Entrypoints = []EntrypointName{AddChainName, AddPreChainName, GetSTHName, GetSTHConsistencyName, GetProofByHashName, GetEntriesName, GetRootsName, GetEntryAndProofName}
+var (
+	// Entrypoints is a list of entrypoint names as exposed in statistics/logging.
+	Entrypoints   = []EntrypointName{AddChainName, AddPreChainName, GetSTHName, GetSTHConsistencyName, GetProofByHashName, GetEntriesName, GetRootsName, GetEntryAndProofName}
+	treeHasher, _ = merkle.Factory(merkle.RFC6962SHA256Type)
```

Comment:
nit: Move the merkle.Factory() call to a `getDefaultHasher` method and have it panic if `err != nil`? Otherwise, if it errors, we won't know until the system starts getting null pointers from treeHasher.